scripts/bash/getInternalDependencies.sh

```
@@ -11,14 +12,36 @@ fi
+    'expression-parser' \
+    'indicators' \
+    'weather-api' \
+    'kobo-api' \
+    'superset-api' \
+    'data-broker' \
+    'server-boilerplate'
   exit 0
 fi
 
 # we are getting internal dependencies for a specific package.json
 internal_dependencies=($(sed -n '/"dependencies": {/,/}/p' ${PWD}/${package_path}/package.json | grep -o '@tupaia/[^"]*": "[0-9\.]*"' | cut -d / -f 2 | cut -d \" -f 1))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)